projects/rocprofiler-systems/source/lib/rocprof-sys/library/amd_smi.cpp

```diff
@@ -241,6 +333,101 @@ get_state()
     static std::atomic<State> _v{ State::PreInit };
     return _v;
 }
+
+std::vector<uint8_t>
+serialize_xcp_metrics(const bool& is_vcn_supported, const bool& is_jpeg_supported,
```

Comment:
```suggestion
serialize_xcp_metrics(const bool& use_vcn_activity, const bool& use_jpeg_activity,
```
Correct, it's just that the variable names are a little confusing because you don't really mean "is VCN supported", you're determining if you should use the "vcn_activity" or "vcn_busy" entries to collect the engine activity data.